ml/src/test/scala/ai/h2o/sparkling/ml/PipelinePredictionTest.scala

```diff
@@ -31,7 +31,6 @@ import org.apache.spark.{SparkContext, SparkFiles}
 import org.junit.runner.RunWith
 import org.scalatest.FunSuite
 import org.scalatest.junit.JUnitRunner
-import water.api.TestUtils
 
 abstract class PipelinePredictionTestBase extends FunSuite with SharedH2OTestContext {
```

Comment:
nit: Could you rename the file to match the conventions: `PipelinePredictionTest` -> `PipelinePredictionTestSuite`?

Reply:
Will do